apps/desktop/desktop_native/windows_plugin_authenticator/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "windows_plugin_authenticator"
+version = "0.0.0"
+edition = "2021"
+license = "GPL-3.0"
+publish = false
+
+[target.'cfg(target_os = "windows")'.build-dependencies]
+bindgen = "0.71.1"
+
+[target.'cfg(windows)'.dependencies]
+windows = { version = "0.59.0", features = ["Win32_Foundation", "Win32_Security", "Win32_System_Com", "Win32_System_LibraryLoader" ] }
+windows-core = "0.59.0"
+ciborium = "0.2.2"
+hex = "0.4.3"
```

We should be using `{ workspace = true }` to avoid duplicating data, and defining the dependency versions in the workspace Cargo.toml: https://github.com/bitwarden/clients/pull/13750
```suggestion
[package]
name = "windows_plugin_authenticator"
version = { workspace = true }
edition = { workspace = true }
license = { workspace = true }
publish = { workspace = true }

[target.'cfg(target_os = "windows")'.build-dependencies]
bindgen = { workspace = true }

[target.'cfg(windows)'.dependencies]
windows = { workspace = true, features = ["Win32_Foundation", "Win32_Security", "Win32_System_Com", "Win32_System_LibraryLoader" ] }
windows-core = { workspace = true }
ciborium = { workspace = true }
hex = { workspace = true }
```

---

Thanks - updated in https://github.com/bitwarden/clients/pull/13278/commits/1ab8f96d7ec0b2c3016f57236ccf1282d4e6ae4c

---

Blocked waiting for this: https://github.com/bitwarden/clients/pull/13974

---

Updated in: https://github.com/bitwarden/clients/pull/13278/commits/925b4615574363f19a67c360490ffdc0148ad8c6